### PR TITLE
removes unnecessary #fetchUrl call

### PR DESCRIPTION
**Pull request by aars**
Found this redundant #fetchUrl call when researching it's usage after [this comment/issue](https://github.com/pixelhandler/ember-jsonapi-resources/pull/96#discussion_r77786086)

### CODE REVIEW

File: addon/adapters/application.js [hunk 1]
```diff
@@ -128,7 +128,6 @@ export default Ember.Object.extend(FetchMixin, Evented, {
     // use resource's service if in container, otherwise use this service to fetch
     let owner = (typeof getOwner === 'function') ? getOwner(this) : this.container;
     let service = owner.lookup('service:' + pluralize(type)) || this;
-    url = this.fetchUrl(url);
```
Comment (pixelhandler) [inline addon/adapters/application.js]: @aars I'm pretty sure this is the hook I use to be sure that when a proxy server is in use the url in the payload for the relationship is replaced with the url of the proxy server.

Comment (aars) [inline addon/adapters/application.js]: I'm pretty sure [this](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/adapters/application.js#L356) is that hook.

Comment (pixelhandler) [inline addon/adapters/application.js]: @aars I'll need to step trough the call stack for a request to updateRelationship (PATCH) I think that relationship requests do not use adapter.fetch.

Comment (aars) [inline addon/adapters/application.js]: (updateRelationship calls patchRelationship](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/mixins/resource-operations.js#L136) which calls [fetch](https://github.com/pixelhandler/ember-jsonapi-resources/blob/master/addon/adapters/application.js#L281)

Even if updateRelationship's stack did not call `fetch`, I removed `fetchUrl` from `findRelated`, which is unrelated to `updateRelationship` or `patchRelationship`. 

But do check to be sure.

**Pull request closed without merging**